cmd/openshift-install/create.go

```diff
@@ -445,12 +445,18 @@ func waitForConsole(ctx context.Context, config *rest.Config) (string, error) {
 			for _, route := range consoleRoutes.Items {
```

Comment:
qq: so we know the route's name today, but still we use List and range over the items to find out route resource, that seems like an overkill. @wking wdyt changing this to Get + check for not found ??

Reply:
Done with ad3f7cda8 -> eb343b19d.